cluster/saltbase/salt/etcd/etcd.manifest

```diff
@@ -0,0 +1,54 @@
+{% set ipv4_addr = salt['grains.get']('fqdn_ip4')[0] -%}
+
+{
+"apiVersion": "v1beta3",
+"kind": "Pod",
+"metadata": {"name":"etcd-server"},
+"spec":{
+"hostNetwork": true,
+"containers":[
+    {
+    "name": "etcd-container",
+    "image": "kubernetes/etcd:2.0.5.1",
```

Comment:
This shouldn't be in ContainerVM image since we still support other internal users.